OrbitClientModel/CMakeLists.txt

```diff
@@ -32,7 +32,8 @@ target_compile_options(OrbitClientModelTests PRIVATE ${STRICT_COMPILE_FLAGS})
 
 
 target_sources(OrbitClientModelTests PRIVATE
-               CaptureDeserializerTest.cpp)
+        CaptureDeserializerTest.cpp
```

Comment:
We have a bit of a mix of indentation there, but it should usually be like below, shouldn't it?
```
target_sources(
  OrbitClientModelTests
  PRIVATE CaptureDeserializerTest.cpp
          CaptureSerializerTest.cpp)
```

Reply:
Well actually I think the one below was wrong. At least when looking at the formatting in other recent CMakeLists files.
We should really add clang-format checks for the cmake files though.

Reply:
Ok, so apparently we have what I suggested mainly in OrbitQt and OrbitGl, which are the only ones I was looking at for reference... nvm then, but agree, clang-format for this would be nice, we shouldn't need to care about this, neither when writing nor when reviewing code.

Reply:
Agree.